client/src/components/Posts/Card.tsx

```diff
@@ -12,11 +15,16 @@ const Card: React.FC<CardProps> = ({
   filename,
 }: CardProps) => {
   return (
-    <div className="flex items-center border-2 border-black px-5 py-3 my-2 w-96">
-      <a href={link}>
+    <div className="flex items-center flex-col border-2 border-black px-5 py-1 my-3 w-80 ">
+      {/* <a href={link}>
```

Comment:
Lets keep option for both viewing/listening to the file and downloading it at the same time.